eng/scripts/Language-Settings.ps1

```diff
@@ -560,16 +560,8 @@ function Import-Dev-Cert-python
   python $pathToScript
 }
 
-function Validate-Python-DocMsPackages
+function Validate-Python-DocMsPackages ($PackageInfo, $PackageSourceOverride, $DocValidationImageId)
 {
-  Param(
-    [Parameter(Mandatory=$true)]
```

Comment:
Just for my understanding was the real problem here the fact this was a mandatory parameter or was there some other underlying issue that required this change?